src/lightning/pytorch/loops/fit_loop.py

```diff
@@ -343,10 +343,8 @@ def advance(self) -> None:
 
         combined_loader = self._combined_loader
         assert combined_loader is not None
-        if combined_loader._mode not in ("max_size_cycle", "min_size"):
-            raise ValueError(
-                f'`{type(self).__name__}` only supports the `CombinedLoader(mode="max_size_cycle" | "min_size")` modes.'
-            )
+        if combined_loader._mode == "sequential":
+            raise ValueError(f'`{type(self).__name__}` does not support the `CombinedLoader(mode="sequential")` mode.')
```

Comment:
@carmocca The previous message was better, because it at least hinted at what the user could try otherwise. Now, the error message just says what doesn't work and has no further info

Reply:
Added suggestion!